plugins/digitalocean/test/test_collector.py

```diff
@@ -0,0 +1,345 @@
+from resoto_plugin_digitalocean.collector import DigitalOceanTeamCollector
+from resoto_plugin_digitalocean.resources import DigitalOceanTeam
+from fixtures import (
+    droplets,
+    regions,
+    volumes,
+    vpcs,
+    databases,
+    k8s,
+    snapshots,
+    load_balancers,
+    floating_ips,
+    projects,
+    project_resources,
+)
+from resotolib.graph import sanitize
+from resotolib.baseresources import Cloud
+from resotolib.graph import Graph, GraphRoot
+import datetime
+
+
+class ClientMock(object):
+    def __init__(self, responses):
+        self.responses = responses
+
+    def __getattr__(self, name):
+        def wrapper(*args, **kwargs):
+            return self.responses.get(name, [])
+
+        return wrapper
+
+
+def prepare_graph(do_client) -> Graph:
+    cloud = Cloud("do")
+    team = DigitalOceanTeam(id="do:team:test_team")
+    plugin_instance = DigitalOceanTeamCollector(team, do_client)
+    plugin_instance.collect()
+    cloud_graph = Graph(root=cloud)
+    graph = Graph(root=GraphRoot("root", {}))
+    cloud_graph.merge(plugin_instance.graph)
+    graph.merge(cloud_graph)
+    sanitize(graph)
+    return graph
+
+
+def check_edges(graph: Graph, from_id: str, to_id: str) -> None:
+    for (node_from, node_to, edge) in graph.edges:
+        if node_from.id == from_id and node_to.id == to_id:
+            return
```

Comment:
consider to use `graph.has_edge()`

Reply:
Ah - yes the collector uses the whole node as node id - not ideal from my pov. But yes - I see why.
Maybe add a comment, so other people understand why.